UPGRADE.md

```diff
@@ -53,6 +53,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta1 to 7.0.0-beta2]
 ### [shopsys/project-base]
+- [#533 main php-fpm container now uses multi-stage build feature](https://github.com/shopsys/shopsys/pull/533)
```

Comment:
@TomasLudvik upgrade notes have to be in section "unreleased", please fix it
CC @PetrHeinz

Reply:
My bad, fixed in #542, can someone approve & merge?